lib/functions/versions.bash

```diff
@@ -64,7 +64,7 @@ version_command() {
     sed -i.bak -e "s|^$plugin_name .*$|$plugin_name ${resolved_versions[*]}|" "$file"
     rm -f "$file".bak
   else
-    # Add a trailing newline at the end of the file if missing and file present
```

Comment:
This change could probably be reverted.

Reply:
Oops I don't think I intentionally removed that